<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountMainWindow</class>
 <widget class="QMainWindow" name="AccountMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1167</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1167</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/illust.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labe_log">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>168</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>← Back</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>1091</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/background.png);
font: 10pt &quot;黑体&quot;;</string>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>用户名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>用户类型</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>借阅列表</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>650</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>注销账户</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
